.vscodeignore

```diff
@@ -0,0 +1,5 @@
+.vscode
```

Comment:
There are more things that should be in the `.vscodeignore`. Basically everything that we don't want to be packaged. Please see other projects like hana explorer and also go over all the files and folders in this repo and add everything that should not be packaged.

Reply:
updated